### ISSUE TITLE
Mock up Contact page

### ISSUE THREAD

**Issue by melchoyce** [GSA-TTS/FAC#766]
Breakout issue of #377.

This issue covers the design of the future FAC Contact page.

### Acceptance criteria (We'll know we're done when...)

- [ ] Users can locate where to get in touch with the FAC with questions.
  
### Tasks

- [x] Design
- [ ] Peer review
- [ ] Write implementation notes

### Definition of Done

- [ ] Screens in Figma are ready to be handed off to front end
- [x] ~All text has a contrast ratio of 4.5:1 with the background~ (Built into USWDS)
- [x] Interaction are documented if they deviate from USWDS components
- [x] The needs of someone navigating the site with the use of a screenreader or keyboard has been considered and documented 
- [x] ~There is an option for multilingual support~ (We've established multilingual support in the header)
- [x] There is supporting documentation for alt text, labels for links (anchor + target), error text in forms, tab order, focus order, and what assistive technologies should announce (if applicable)

**Comment by melchoyce** [comment]
I worked on the contact page design with @Leighdiddy and @lauraherring:

![image](https://user-images.githubusercontent.com/2846578/216672972-1e445139-f080-4c0b-81a9-094b9b60a148.png)
[Figma](https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground?node-id=5725%3A33351&t=Wuq6OT3Qgwxucjds-4)

Needs final copy still. Pinging the above folks + @austinhernandez @thebestsophist for peer review.

**Comment by melchoyce** [comment]
Chatted about this during our design backlog chat today. Putting it in our backlog to revisit and update once we have a chance to review the copy for these pages.

**Comment by jadudm** [comment]
We now have a ticketing/help desk system. This work will be picked up as part of those milestones. Closing.
